src/test/java/com/happeningnow/HappeningnowApplicationTests.java

```diff
@@ -1,13 +1,8 @@
 package com.happeningnow;
 
-import org.junit.jupiter.api.Test;
 import org.springframework.boot.test.context.SpringBootTest;
 
+
 @SpringBootTest
 class HappeningnowApplicationTests {
-
-	@Test
-	void contextLoads() {
-	}
-
-}
+}
```

Comment:
Leave the class as it was, cause it ensures the context of the application is properly loaded. Avoiding the risk of forgetting to define any annotation or property

Reply:
Ok